packages/react-composites/tests/browser/chat/app/index.css

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) Microsoft Corporation. */
+/* Licensed under the MIT license. */
+
+body,
+html,
+.Root {
+  max-width: 100%;
```

Comment:
Still seeing the chat composite going beyond screen height:
![image](https://user-images.githubusercontent.com/2684369/173387390-c1706c81-5892-4d13-bd76-1778feba6455.png)

Instead of `max-height` try just `height`? My guess is because the chat composite sizes itself to 100% of the avilable space of the parent, however html, body and root don't actually have a height so they are considered height: 0.
If that doesn't work feel free to wrap the chat composite in a container div sized to `100vh`